src/app/components/word-list/word-list.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="words-list tile-view">
-  <div *ngFor="let w of words$ | async" class="one-word">
+  <div *ngFor="let w of words$ | async" class="one-word" (click)="wordDetialsModal(w,content)">
```

Comment:
Do you have to add content each time you click? or you can remove it from the function and use it in the code ?

Reply:
fixed 
https://ng-bootstrap.github.io/#/components/modal/examples